projects/angular-cld/src/lib/constants.ts

```diff
@@ -11,9 +11,10 @@ export const predominantColorTransformPxl = [
   {fetch_format: 'auto', quality: 'auto'}];
 
 export const predominantColorTransform  = [
+  {variables: [['$currWidth', 'w'], ['$currHeight', 'h']]},
```

Comment:
@strausr , what are 'h' & 'w'?

Reply:
This is how you provide user variables.. height and width.